src/app/PCM/templates/tpl-detail-view/tpl-property-detail-view/amenity-field.js

```diff
@@ -33,7 +33,7 @@ const AmenityField = ({ attributeCode, code }) => {
   // if a is not null or undefined, foo = a
   // if a is null or undefined, foo = b
   // Unlike ||, which if foo = a || b, and a = falsey value, then foo = b
-  const value = entityAttribute?.value ?? '?'
+  const value = entityAttribute?.valueInteger ?? '?'
```

Comment:
remove all the commented lines from above and do the foloowing instead

  const entityAttributeValueInteger = entityAttribute?.valueInteger
  const entityAttributeValue = entityAttribute?.value

  const value = entityAttributeValue || entityAttributeValueInteger || ''

Reply:
Done.